src/it/scala/com/databricks/spark/redshift/RedshiftReadSuite.scala

```diff
@@ -195,33 +193,42 @@ class RedshiftReadSuite extends IntegrationSuiteBase {
         s"CREATE TABLE $tableName (x real)")
       conn.createStatement().executeUpdate(
         s"INSERT INTO $tableName VALUES ('NaN'), ('Infinity'), ('-Infinity')")
-      conn.commit()
       assert(DefaultJDBCWrapper.tableExists(conn, tableName))
-      // Due to #98, we use Double here instead of float:
       checkAnswer(
         read.option("dbtable", tableName).load(),
-        Seq(Double.NaN, Double.PositiveInfinity, Double.NegativeInfinity).map(x => Row.apply(x)))
+        Seq(Float.NaN, Float.PositiveInfinity, Float.NegativeInfinity).map(x => Row.apply(x)))
```

Comment:
Just curious about how this interacts with the deleted comment on L200. Is that due to a bug that is no longer relevant?

Reply:
Let me ping @dichiarafrancesco here because I'm merging in 17 of his past commits when he worked on Yelp/spark-redshift. I started my 10 commit change on top of Yelp/spark-redshift while this new project was forked from databricks/spark-redshift. The result here is a very big diff.

Reply:
Previously spark-redshift used to cast real type into double (mistakenly since Redshift Real is represented in 4 bytes IEEE Float - https://docs.aws.amazon.com/redshift/latest/dg/r_Numeric_types201.html). Although they spot the bug (issue #98), they didn't fix that because of the incompatibility nature between doubles and floats in Parquet. I actually made the change (https://github.com/spark-redshift-community/spark-redshift/pull/6/files#diff-69806564231efb590460b162532ba683R333) regardless, since we've almost never used the version using doubles. So I think it's fine to drop the comment and fix the test in this way.

Reply:
Thanks for your input @dichiarafrancesco, I'll close this